<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>450</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Чат-клиент</string>
    </property>
    <widget class="QWidget" name="centralwidget">
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
          <layout class="QHBoxLayout" name="connectionLayout">
            <item>
              <widget class="QLabel" name="labelIp">
                <property name="text">
                  <string>IP сервера:</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLineEdit" name="ipLineEdit">
                <property name="placeholderText">
                  <string>127.0.0.1</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="labelName">
                <property name="text">
                  <string>Имя:</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLineEdit" name="nameLineEdit">
                <property name="placeholderText">
                  <string>Ваше имя</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="connectButton">
                <property name="text">
                  <string>Подключиться</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="disconnectButton">
                <property name="enabled">
                  <bool>false</bool>
                </property>
                <property name="text">
                  <string>Отключиться</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>
        <item>
          <widget class="QTextBrowser" name="chatBrowser">
            <property name="minimumSize">
              <size>
                <width>0</width>
                <height>200</height>
              </size>
            </property>
          </widget>
        </item>
        <item>
          <layout class="QHBoxLayout" name="messageLayout">
            <item>
              <widget class="QComboBox" name="userComboBox">
                <property name="enabled">
                  <bool>false</bool>
                </property>
                <item>
                  <property name="text">
                    <string>Все</string>
                  </property>
                </item>
              </widget>
            </item>
            <item>
              <widget class="QLineEdit" name="messageLineEdit">
                <property name="enabled">
                  <bool>false</bool>
                </property>
                <property name="placeholderText">
                  <string>Сообщение...</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="sendButton">
                <property name="enabled">
                  <bool>false</bool>
                </property>
                <property name="text">
                  <string>Отправить</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>
      </layout>
    </widget>
    <widget class="QMenuBar" name="menubar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>600</width>
          <height>22</height>
        </rect>
      </property>
    </widget>
    <widget class="QStatusBar" name="statusbar"/>
  </widget>
  <resources/>
  <connections/>
</ui>
